### PR TITLE
Update DOCUMENTATION.org

### PR DESCRIPTION
**Pull request by treymerkley**
I removed the 'slant and replaced it with slant, which is more accurate.

### CODE REVIEW

**Comment by gcentauri** [comment]
I can confirm that if you use a quote for the separator type it will break the modeline with this error 

```
Error during redisplay: (eval (spaceline-ml-all-the-icons)) signaled (error "Unable to find icon with name ‘(quote slant)-right’ in icon set ‘alltheicon’") [11 times]
```

**Comment by gcentauri** [comment]
@treymerkley - you can also add your change to the CHANGELOG.develop file to make it easier for maintainers if you want to.

**Comment by treymerkley** [comment]
> @treymerkley - you can also add your change to the CHANGELOG.develop file to make it easier for maintainers if you want to.

That *should* be done now. Thanks!

File: CHANGELOG.develop [hunk 1]
```diff
@@ -2821,7 +2821,9 @@ Other:
   Stewart, timor, TinySong, Titov Andrey, Thomas de Beauchêne, Tomasz
   Cichocinski, tzhao11, Vincent Taing, Vlad Bokov, Vladimir Kochnev, Wieland
   Hoffmann, Witoslaw Koczewski, Xiang Ji, Yi Liu, Zane Sterling, zer09, Zhige
-  Xin, Serghei Iakovlev)
+  Xin, Serghei Iakovlev)  
+**** Documentation and website
```
Comment (sdwolfz) [inline CHANGELOG.develop]: Right above this section there is `Various documentation improvements` with a list of all people that have contributed to the docs. Please add your name in that list if it does not already exist.

Comment (treymerkley) [reply CHANGELOG.develop]: done, thanks! I'll squash these in a few minutes.

Comment (treymerkley) [reply CHANGELOG.develop]: _two days after the initial squash request_, squashed.

**Comment by duianto** [comment]
Thank you for contributing to Spacemacs. It's safe to delete your branch.
The PR has been cherry picked into the `develop` branch. https://github.com/syl20bnr/spacemacs/commit/1646a246d614008c82323a5184a37c09ff8b5bdb

**Comment by duianto** [comment]
A note for your next PR, the commit message had a couple of duplicate lines and there was two trailing spaces after the line:
`+  Xin, Serghei Iakovlev)  `

emacs_2019-09-21_21-33-41

You can rewrite the last commit with `c w` for (commit, reword) in magit.
It's also possible to both apply a change to the commit and reword its message with
`c a` for (commit, amend).

**Pull request closed without merging**